README.md

```diff
@@ -1,24 +1,119 @@
-# README
+# Private-Events
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Table of Contents
+  <br />
 
-Things you may want to cover:
+* [About the Project](#about-the-project)
+* [Built With](#built-with)
+* [Contributing](#contributing)
+* [Contact](#authors)
+* [Acknowledgements](#acknowledgements) 
 
-* Ruby version
+#
+<br />
 
-* System dependencies
+[![forthebadge](https://forthebadge.com/images/badges/made-with-ruby.svg)](https://forthebadge.com) </br>
+![rails](https://img.shields.io/badge/Ruby_on_Rails-CC0000?style=for-the-badge&logo=ruby-on-rails&logoColor=white)
 
-* Configuration
+<br />
 
-* Database creation
+#
 
-* Database initialization
 
-* How to run the test suite
+<!-- ABOUT THE PROJECT   -->
+## About The Project
 
-* Services (job queues, cache servers, search engines, etc.)
+The Private Event project sets up an initial rails app containing a User, Event databases. This page can log in users and create events for each of them. As well can let users attend others events.
+
+## Screenshot Test
+#
+<p align="center">
+  <img height="auto" src="Screenshot.png">
+</p>
+
+#
+
+### Built With
+
+* [RUBY]
+* [RAILS]
+* [DEVISE]
+* [RSPEC&CAPIBARA]
+
+## Contributing
```

Comment:
- This section should be towards the end of this page. Add it before the `Acknowledgements` section.